app/1.0/docs/devguide/data-system.md

```diff
@@ -326,16 +326,16 @@ Related tasks:
 Polymer implements the mediator pattern, where a host element manages data flow between itself and
 its local DOM nodes.
 
-When two elements are connected with a data binding, data changes can flow *downward*, that is, from
-host to target, *upward*, from target to host, or both ways.
+When two elements are connected with a data binding, data changes can flow **downward** — from
+host to target — **, upward** — from target to host — , or both.
 
-When two elements in the local DOM are bound to the same property, data appears to flow from one
-element to the other, but this flow is *mediated* by the host. A change made by one element
-propagates *up* to the host, then the host propagates the change *down* to the second element.
+When two elements in the local DOM are bound to the same property data appears to flow from one
+element to the other, but this flow is **mediated** by the host. A change made by one element
```

Comment:
Technically, we're introducing a new term here, so mediated should be italic. In practice, we don't yet have a good definition in place, and the reader is left to infer the definition from the following text. Leave it italics or remove the emphasis, but don't make it bold and call extra attention to my failure here :D